tools/lou_trace.c

```diff
@@ -112,7 +112,7 @@ print_number(widechar c) {
 }
```

Comment:
Why is this change part of this PR? Isn't it already part of #1062?

Reply:
It was part of this PR first. Will be solved when this is rebase onto the other.

Reply:
OK, fine with me